ext/ExaPFCUDAExt.jl

```diff
@@ -0,0 +1,30 @@
+module ExaPFCUDAExt
+
+using ExaPF
+using CUDA
+# import CUDA.CUBLAS
+# import CUDA.CUSPARSE: CuSparseMatrixCSR, CuSparseMatrixCSC
+# import CUDA.CUSOLVER
+using CUDA.CUBLAS
+using CUDA.CUSOLVER
+using CUDA.CUSPARSE
+
+using ForwardDiff
+using LinearAlgebra
+using KernelAbstractions
+using SparseArrays
+
+const KA = KernelAbstractions
+const LS = ExaPF.LinearSolvers
+const PS = ExaPF.PowerSystem
+const AD = ExaPF.AutoDiff
+
+ExaPF.LS.DirectSolver(J::CUSPARSE.CuSparseMatrixCSR; options...) = ExaPF.LS.DirectSolver(nothing)
```

Comment:
maybe remove `ExaPF.` here, as `LS` is already defined on line 18? Same on the following lines